trustchain-api/src/api.rs

```diff
@@ -444,4 +573,16 @@ mod tests {
             .await
             .unwrap()
     }
+
+    #[test]
+    fn get_key_entry() {
+        use ps_sig::keys::Params;
+        use ssi::jwk::rss::generate_keys_jwk;
+        use ssi::jwk::JWK;
+
+        let key: JWK = generate_keys_jwk(64, &Params::new("test".to_string().as_bytes())).unwrap();
+        println!("{}", serde_json::to_string_pretty(&key).unwrap());
+        let entry: PublicKeyEntry = key.try_into().unwrap();
+        println!("{}", serde_json::to_string_pretty(&entry).unwrap());
+    }
```

Comment:
Refer to this test in the Dev Guide somewhere? It's used to generate the key entry format put onto IPFS for an expandable public key.